soql-server-pg/src/main/scala/com/socrata/pg/query/DataSqlizerQuerier.scala

```diff
@@ -69,8 +71,23 @@ trait DataSqlizerQuerier[CT, CV] extends AbstractRepBasedDataSqlizer[CT, CV] wit
     row.freeze()
   }
 
-  private def executeSql(conn: Connection, pSql: ParametricSql): ResultSet = {
+  def execute(conn: Connection, s: String) {
+    using(conn.createStatement()) { stmt =>
+      logger.trace("Executing simple SQL {}", s)
+      stmt.execute(s)
+    }
+  }
+
+  def setTimeout(timeoutMs: String) = s"SET LOCAL statement_timeout TO $timeoutMs"
+  def resetTimeout = "SET LOCAL statement_timeout TO DEFAULT"
+
+  private def executeSql(conn: Connection, pSql: ParametricSql, timeout: Option[Duration]): ResultSet = {
     try {
+      if (timeout.isDefined && timeout.get.isFinite()) {
```

Comment:
is not infinite or NaN :) (also cause that seems to be our common check for "hey is this a real thing", sprinkled through code, but mostly the first bit)